code/src/components/Thoughts.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import moment from 'moment'
+
+import { Heart } from './Heart'
+
+import './thoughts.css'
+
+/* This thought-component takes care of showing all messages that has been posted to the API.
+It uses a state that changes based on what the data from fetch is. */
+export const Thoughts = ({ thoughts, likeMessage }) => {
+
+    return (
+        <div className="thoughts__container">{thoughts.filter((thought) => thought.message !== undefined).map((thought) => {
+            return (
+                <div className="thoughts__card" key={thought._id}>
+                    <div>
+                        <p className="thoughts__message">{thought.message}</p>
```

Comment:
A bit more styling is needed, right now your website broke a bit when I wrote super long word without any spaces